services/static-webserver/client/source/class/osparc/utils/LibVersions.js

```diff
@@ -55,7 +55,7 @@ qx.Class.define("osparc.utils.LibVersions", {
 
       let url = remoteUrl;
       if (commitId) {
-        url = remoteUrl + "/tree/" + String(commitId) + "/services/web/client/";
+        url = remoteUrl + "/tree/" + String(commitId) + "/services/static-webserver/client/";
```

Comment:
not sure why this public url had to change, why call it static-web server and not leave it just as web?

Reply:
this is the path to the file. since all the client code was moved to the static-webserver that is the new path.